apps/src/weblab/brambleHost.js

```diff
@@ -14,11 +14,29 @@ window.requirejs.config({baseUrl: BRAMBLE_BASE_URL});
 let webLab_;
 if (parent.getWebLab) {
   webLab_ = parent.getWebLab();
-} else {
+} else if (!brambleConfig.skipFiles) {
   console.error('ERROR: getWebLab() method not found on parent');
 }
 
 function load(Bramble) {
+  if (brambleConfig.skipFiles) {
```

Comment:
I'm a bit concerned that future maintainers will get a bit turned around by the "skipFiles" double negative above and the branching condition here. Given that this is a special case and the code is entirely separate from the rest of the load function, it may make more sense to move this into its own function to keep an unfortunate future curly brace from causing the functionality to bleed between the two.

For example:
`function load(Bramble) // normal method`
`function loadMinimal(Bramble) // support check method`

Reply:
Then above, the `else` could be something like
```
else {
  brambleConfig.skipFiles || console.error('ERROR: getWebLab() method not found on parent');
}
```

Reply:
That's funny -- I had what you had initially, but then I thought it might also confuse future maintainers to have branching logic in the line `requirejs([brambleClient], load);` at the bottom of this file. I'm happy to change it back, though.

Reply:
Ha. Well I'd defer to you then. May just be a matter of opinion in that case.

Reply:
What do you think of this? Ultimately I think I agree that I like it better! https://github.com/code-dot-org/code-dot-org/blob/76c8ef3ecb943b610f28ac2281ddba0c81c9ca1f/apps/src/weblab/brambleHost.js